### DISCUSSION TITLE
Cant connect github with code-server

### DISCUSSION THREAD

**Discussion by Default-01** [coder/code-server#5030] [Q&A] [answered]
Im unable to authorize the github extention with code-server. Everytime when im getting redirected to logit with github i get this error.

![5CA7A771-3E48-4723-BE78-D89499D5B6C8](https://user-images.githubusercontent.com/55616945/160079377-0f870d16-5c34-47c8-9139-dfe63f121169.png)

i am using railway as host

**Comment by jsjoeio** [comment]
Which version of code-server are your running?

**Comment by Default-01** [comment, marked as answer]
im using the latest version

**Reply by jsjoeio** [reply]
can you post steps to reproduce? I can try locally

**Reply by Default-01** [reply]
for some reason it now works, though thank you for the help

**Reply by jsjoeio** [reply]
Oh wow, great!
